assets/airports/kstl.json

```diff
@@ -1321,7 +1321,7 @@
             "origin": "kstl",
             "destination": "",
             "category": "departure",
-            "route": "KSTL.TURBO9.GENTS",
+            "route": "KSTL30L.TURBO9.GENTS",
```

Comment:
29 or 30L? It would make sense to be 30L if you have to fly round to CSX, but then GENTS is to the south so it would make sense to use 29 if you are going directly onto the transition (ie. direct LEEAN)